apiary.apib

```diff
@@ -50,6 +51,7 @@ API endpoints that Operation Code's Rails backend makes available to its React f
     + has_online (boolean, required) - Does the CodeSchool have an online component
     + online_only (boolean, required) - Is the CodeSchool online only?
     + notes (text, optional) - Any additional information about the CodeSchool
+    + mooc (boolean, required) - Does the CodeSchool offer Massive Open Online Courses?
```

Comment:
These will actually be `optional`.  Since we are setting a default of `false` at the db level, this is already accounted for.